assets/src/edit-story/components/form/numeric.js

```diff
@@ -83,6 +83,35 @@ function Numeric({
   const [dot, setDot] = useState(false);
   const ref = useRef();
 
+  const altPressed =
+    float && useIsKeyPressed(ref, { key: 'alt', editable: true });
+
+  const handleUpDown = useCallback(
+    ({ key }) => {
```

Comment:
If we'd update `createKeyHandler` as mentioned above, we'd get the sequence like `alt+up` as the second argument, which would be very convenient.

Or `createKeyHandler` could even pass the sequence from Mousetrap through `resolveKeySpec`